app/services/partnerships/report.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
```

Comment:
Nice!

Reply:
Yeah, it was nightmare to write a test for the whole iterating form! :P